app/helpers/dropdown_helper.rb

```diff
@@ -59,6 +59,7 @@ def document_type_options_for_select(selected = nil)
   def allocation_options_for_select(selected = nil)
     auditors = Audits::FindTeamAuditors
                  .call(user_uid: current_user.uid)
+                 .sort_by(&:name)
```

Comment:
`Audits::FindTeamAuditors` has `.order(name: :asc)` in it as well.

Reply:
Addressed in d72cd9d